simplesource-command-kafka/src/main/java/io/simplesource/kafka/internal/streams/topologies/AggregateStreamResourceNames.java

```diff
@@ -0,0 +1,26 @@
+package io.simplesource.kafka.internal.streams.topologies;
+
+import io.simplesource.kafka.api.AggregateResources;
+import io.simplesource.kafka.api.ResourceNamingStrategy;
+
+public class AggregateStreamResourceNames {
+    private final ResourceNamingStrategy resourceNamingStrategy;
+    private final String aggregateName;
+
+    public AggregateStreamResourceNames(ResourceNamingStrategy resourceNamingStrategy, String aggregateName) {
```

Comment:
If we extract `topicName` and `stateStoreName` methods into a separate class, please can we remove the corresponding methods (and their usages) from the `EventSourcedTopology` class.

Otherwise we're stuck with two ways of doing the same thing.

Reply:
Yes, once I finish with all pieces, all these methods will be removed and only defined in `AggregateStreamResourceNames`